### PR TITLE
release: 0.8.0

### PR DESCRIPTION
**Pull request by mvidalgarcia**

### CODE REVIEW

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:17]: This should be "changes", i.e. not a new thing, but an upgrade from CWL version N1 to CWL version N2.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:7]: Please explain a bit, e.g. "to retrieve information about total CPU and Disk usage and quota limits".

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:9]: Aren't these better "Changes `list` command to add a possibility to filter..." etc?  I.e. not a new command, but changed behaviour via new options.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:13]: Ditto here, perhaps "Changes `logs` command to add start and finish...".  Basically, the structure of the news could be "adds/changes/fixes command foo", and then explaining how.  This will allow to sort release notes news items by command, so  that users will find more easily what's new.

File: CHANGES.rst
Comment (mvidalgarcia) [reply CHANGES.rst:17]: Actually, it should be "Upgrades", I don't really understand this policy of just keeping "Adds/Changes/Fixes/Deletes".

**Comment by mvidalgarcia** [comment]
Updated.

File: CHANGES.rst
Comment (tiborsimko) [reply CHANGES.rst]: It is just a way to represent the traditional "New features / Changed behaviour / Bug fixes / Deprecated" division of news items in the release notes, using common language structure rather than introducing one more list level indentation in 
the RST used elsewhere e.g. in https://www.sphinx-doc.org/en/master/changes.html#release-4-0-0-released-may-09-2021

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:8]: - Adds new command ``info``  that retrieves general information about the cluster, such as available workspace path settings.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:9]: - Changes ``validate`` command to add the possibility to check the workflow against server capabilities such as desired workspace path via `--server-capabilities` option.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:10]: - Adds support for ``outputs.directories`` in ``reana.yaml`` allowing to easily download output directories.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:11]: - Adds support for running and validating Snakemake workflows.

(I guess you can make it new number one in the list, as this is the biggest new feature!) Then the output directories, and then the rest of items will use the usual adds new comand, changes foo command stucture.)

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:14]: This misses continuation... We can say:

- Changes ``list`` command to stop including workflow progress and workspace size by default. Please use new optinos `--include-progress` and `--include-workspace-size` to show this information.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:15]: - Changes ``list --sessions`` command to display the status of interactive sessions.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:16]: - Changes ``logs`` command to display also the start and finish times of individual jobs.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:18]: Changes ``ls`` command to add the possibility to filter by file name, size and last-modified values via ``--filter`` option.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:20]: - Changes ``delete`` command to prevent hard-deletion of workflows.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:23]: - Removes support for Python 2.7. Please use Python 3.6 or higher from now on.

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:22]: - Changes CWL workflow engine to ``cwltool`` version ``3.1.20210628163208``.

(so that it'll play nicely with the Yadage news item above)

File: CHANGES.rst
Comment (tiborsimko) [inline CHANGES.rst:17]: (beware, misplaced line)

**Comment by mvidalgarcia** [comment]
Updated.

File: CHANGES.rst
Comment (matthewfeickert) [reply CHANGES.rst]: > Please use Python 3.6 or higher from now on.

Can you please implement this in `setup.py` with `python_requires`? If support for Python 2 is dropped without a `python_requires>=3.6` in `setup.py` then if a Python 2 user attempts to install `reana-client` from PyPI they won't get the last Python 2 supported release but rather a broken (for Python 2) release.

I guess I could have made this PR in the time it has taken me to write this, so I'll do so now.

**Review by tiborsimko — APPROVED** [review]
Looks good :+1: We may want to merge #581 first.

**Comment by mvidalgarcia** [comment]
Rebased and release date updated. Waiting for the green light to merge and release.

**Pull request merged**
